tools/python/mwm/mwm.py

```diff
@@ -0,0 +1,428 @@
+# MWM Reader Module
+import struct
+import math
+from datetime import datetime
+
+# Unprocessed sections: geomN, trgN, idx, sdx (search index), addr (search address), offs (feature offsets - succinct)
+# Routing sections: mercedes (matrix), daewoo (edge data), infinity (edge id), skoda (shortcuts), chrysler (cross context), ftseg, node2ftseg
+# (these mostly are succinct structures, except chrysler and node2ftseg, so no use trying to load them here)
+
+# TODO:
+# - Predictive reading of LineStrings
+# - Find why polygon geometry is incorrect in iter_features()
+# - Find feature ids in the 'dat' section, or find a way to read the 'offs' section
+
+class MWM:
+    languages = ["default",
+                 "en", "ja", "fr", "ko_rm", "ar", "de", "int_name", "ru", "sv", "zh", "fi", "be", "ka", "ko",
+                 "he", "nl", "ga", "ja_rm", "el", "it", "es", "zh_pinyin", "th", "cy", "sr", "uk", "ca", "hu",
+                 "hsb", "eu", "fa", "br", "pl", "hy", "kn", "sl", "ro", "sq", "am", "fy", "cs", "gd", "sk",
+                 "af", "ja_kana", "lb", "pt", "hr", "fur", "vi", "tr", "bg", "eo", "lt", "la", "kk", "gsw",
+                 "et", "ku", "mn", "mk", "lv", "hi"]
+
+    metadata = ["0",
+                "cuisine", "open_hours", "phone_number", "fax_number", "stars",
+                "operator", "url", "website", "internet", "ele",
+                "turn_lanes", "turn_lanes_forward", "turn_lanes_backward", "email", "postcode",
+                "wikipedia", "maxspeed", "flats", "height", "min_height",
+                "denomination", "building_levels"]
+
+    def __init__(self, f):
+        self.f = f
+        self.coord_size = None
+        self.base_point = (0, 0)
+        self.read_info()
+        self.type_mapping = []
+
+    def read_types(self, filename):
+        with open(filename, 'r') as ft:
+            for line in ft:
+                if len(line.strip()) > 0:
+                    self.type_mapping.append(line.strip())
+
+    def read_info(self):
+        self.f.seek(0)
+        self.f.seek(self.read_uint(8))
+        cnt = self.read_varuint()
+        self.tags = {}
+        for i in range(cnt):
+            name = self.read_string(True)
+            offset = self.read_varuint()
+            length = self.read_varuint()
+            self.tags[name] = (offset, length)
```

Comment:
And add another import just for that? I don't want to. These values are supposed to be internal after all, since end-user methods are `read_whatever` which decodes section contents.